docs/source/API/core/parallel-dispatch/ParallelScanTag.rst

```diff
@@ -0,0 +1,50 @@
+``ParallelScanTag()``
+=====================
+
+.. role::cpp(code)
+    :language: cpp
+
+Header File: ``<Kokkos_ExecPolicy.hpp>``
+
+.. _text: ../parallel-dispatch/parallel_scan.html
+
+.. |text| replace:: ``parallel_scan``
+
+A tag used in team size calculation functions to indicate that the functor for which a team size is being requested is being used in a |text|_
```

Comment:
May be easier to do a cross-reference here

Reply:
Is something like this appropriate?

Code:
![image](https://user-images.githubusercontent.com/62652182/214420772-8a446ed4-d2e6-4cac-bc15-5df0ef00a3b7.png)

Render:
![image](https://user-images.githubusercontent.com/62652182/214420887-e0208000-f6b8-4d80-b6ec-de297649a3ef.png)

The writing is complex because the word must have 1, the property to be clickable towards an external link and 2, to be in blue so clickable.

The improvement with your advice is the appearance of the brackets! If this rendering suits you I will apply it to the other issues. @nmm0

Reply:
If this is not the way you expected to do it, I'd like some help: I've tried many things and despite the documentation, I can't manage to simplify this writing (if it's possible). @nmm0